Horion/Module/Modules/TriggerBot.cpp

```diff
@@ -16,20 +17,35 @@ const char* TriggerBot::getModuleName()
 {
 	return ("TriggerBot");
 }
-
 void TriggerBot::onTick(C_GameMode* gm)
 {
 	C_LocalPlayer* localPlayer = g_Data.getLocalPlayer();
 	C_Entity* target = g_Data.getClientInstance()->getPointerStruct()->entityPtr;
 	Odelay++;
 	if (target != 0 && Odelay >= delay)
 	{
+		if (target == localPlayer) // Skip Local player
+			return;
+
+		if (target == 0)
+			return;
+
+		if (target->timeSinceDeath > 0 || target->damageTime >= 7)
+			return;
+
+		if (FriendList::findPlayer(target->getNameTag()->getText())) // Skip Friend
+			return;
+
+		if (!Target::isValidTarget(target))
+			return;
+
 		if (localPlayer->getEntityTypeId() == target->getEntityTypeId())
 		{
 			localPlayer->swingArm();
 			gm->attack(target);
 		}
-		Odelay = 0;	
+		Odelay = 0;
+		}
```

Comment:
remove this bracket, you dont need it as you removed the else

Reply:
ffs i removed it on VS but forgot to upload

Reply:
adding a closing bracket here and removing one from the bottom isnt a great way to fix the situation